applovin_max/ios/Classes/AppLovinMAXAdViewWidget.m

```diff
@@ -29,6 +29,8 @@ - (instancetype)initWithAdUnitIdentifier:(NSString *)adUnitIdentifier adFormat:(
         self.adView.delegate = self;
         self.adView.revenueDelegate = self;
         
+        [self.adView setExtraParameterForKey: @"adaptive_banner" value: @"true"];
```

Comment:
Should use the new `MAAdViewConfiguration` object to enable adaptive banners (else we end up logging a warning on newer SDKs): https://developers.applovin.com/en/max/ios/ad-formats/banner-and-mrec-ads#anchored-adaptive-banners